scripts/dnd5e-helpers.js

```diff
@@ -816,9 +816,9 @@ Hooks.on("preUpdateActor", async (actor, update, options, userId) => {
   if ((game.settings.get('dnd5e-helpers', 'gwEnable')) && (hp !== undefined)) {
     GreatWound_preUpdateActor(actor, update);
   }
-if((game.settings.get('dnd5e-helpers', 'owHp0')) && (hp === 0)){
-  OpenWounds(actor)
-}
+  if ((game.settings.get('dnd5e-helpers', 'owHp0')) && (hp === 0)) {
+    OpenWounds(actor, "from falling to 0hp")
```

Comment:
in accordance with scope narrowing at line 793

```suggestion
    OpenWounds(actor.data.name, "from falling to 0hp")
```